### PR TITLE
Add maven-wrappers.

### PR DESCRIPTION
**Pull request by motlin**
You can feel free to close this if you don't think it would be useful in classes, but I figured I'd share the maven wrappers and show how they can be used.

First, I added the wrappers by running `mvn -N io.takari:maven:wrapper -Dmaven=3.6.3` which created `.mvn/wrapper`, `mvnw`, and `mvnw.cmd`. I manually copied `.mvn/jvm.config` and `.mvn/maven.config` from Eclipse Collections. I also added the default goals of `clean install` in the root pom.

With the wrappers in place, you can clone the repo and run `./mvnw` at the root to run the build. You can also cd into a specific kata and run the wrapper just on that subdirectory.

```bash
cd company-kata/
../mvnw
```

### CODE REVIEW

**Comment by nikhilnanivadekar** [comment]
LGTM

**Comment by nikhilnanivadekar** [comment]
Feel free to merge

File: pom.xml [hunk 1]
```diff
@@ -99,6 +99,10 @@
         <maven.compiler.maxmem>2048m</maven.compiler.maxmem>
     </properties>
 
+    <build>
+        <defaultGoal>clean install</defaultGoal>
```
Comment (donraab) [inline pom.xml:103]: Should default goal be `clean test` for the kata?

**Comment by motlin** [comment]
Install includes test. Arguably `verify` would be the best. We can play
around with different choices - I don't think it matters much for the kata.

**Pull request merged**